roles/jd-server/src/lib/job_declarator/message_handler.rs

```diff
@@ -38,6 +41,53 @@ impl JobDeclaratorDownstream {
     }
 }
 
+pub fn clear_declared_mining_job(
+    mining_job: DeclareMiningJob,
+    mempool: Arc<Mutex<JDsMempool>>,
+) -> Result<(), Error> {
+    // If there is an old declared mining job, remove its transactions from the mempool
+    // Retrieve necessary data from the old job
```

Comment:
No need for these comments. Also, the`pub` is redundant.

Reply:
@NonsoAmadi10, The comment is resolved, but I don't see the requested changes.